test/java/seedu/addressbook/logic/AttendanceTest.java

```diff
@@ -101,14 +101,14 @@ public void setUp() throws Exception {
      */
 
     @Test
-    public void executeUpdateAttendanceInvalidArgsFormat() throws Exception {
+    public void execute_updateAttendance_invalidArgsFormat() throws Exception {
```

Comment:
The format for test is
featureUnderTest_testScenario_expectedBehavior().

Reply:
so like updateAttendance_execute_invalidArgsFormat?

Reply:
ok so is :
executeUpdateAttendance_invalidArgs_invalidArgsMessage

Reply:
yes